### ISSUE TITLE
NVIDIA AFX SDK for 4090 and Nvidia Noise Removal does not work in 30.2.2 OBS. 

### ISSUE THREAD

**Issue by Sopheus** [obsproject/obs-studio#11247]
### Operating System Info

Windows 10

### OBS Studio Version

30.2.2

### OBS Studio Log URL

https://obsproject.com/logs/MOtwdnwM51OQQFEy

### Expected Behavior

Nvidia Noise Removal work normally, cutting out background noise.

### Current Behavior

Hardware:
* Microphone: Maono PD200X
* PC: Intel Core i9-13900K, NVIDIA GeForce RTX 4090, 64GB RAM, 2TB SSD

Software:
* OBS Studio 30.2.2
* NVIDIA Broadcast Audio Effects SDK (1.3.0.22) for RTX 4090 (Ada)

Issue Description:
I am encountering a problem with NVIDIA Noise Removal on my RTX 4090 in OBS. Instead of reducing background noise, the filter significantly degrades the audio quality, producing a "talking into a tube" effect. 

Troubleshooting Steps:
* I installed the same version of NVIDIA Broadcast Audio Effects SDK (1.3.0.22) designed for previous GPU generations (Ampere and Turing). Noise Removal functioned correctly with these versions, but only before adjusting the filter settings.
* Once the Noise Removal slider is moved in the filter settings, the filter stops working as intended.
* The only way to restore functionality is to restart OBS.

Current Workaround:
My current workaround involves a repetitive process:

1. Start OBS.
2. Adjust the Noise Removal slider to the desired level.
3. Restart OBS.
4. Check if the noise removal effect is satisfactory.
5. If not, repeat steps 2-4 until the desired effect is achieved.

Given the fact that none of the SDK versions (ADA, AMPERE, TURNING) work properly, my guess it is a OBS issue.

### Steps to Reproduce

1. Install OBS Studio 30.2.2 and NVIDIA Broadcast Audio Effects SDK (1.3.0.22) for RTX 4090 (Ada)
2. Select Nvidia Noise Removal
3. Observe the issue

**Comment by pkviet** [comment]
Thanks for your report. OBS dev who worte the filter here.
I am unable to repro your issue with 30.2.2 and an RTX 4090, same driver version.

https://github.com/user-attachments/assets/a57722c0-c4b2-4f36-992b-0a30aa75aa16

There must be something else interfering in your setup; if i read your log, i see other filters applied on top of noise suppression; could you retest with the other filters disabled ?

**Comment by Sopheus** [comment]
Yes, even with other filters disabled the issue still happens. After you run OBS you need to move the slider of Nvidia Noise Removal while Monitoring is ON.

**Comment by pkviet** [comment]
> Yes, even with other filters disabled the issue still happens. After you run OBS you need to move the slider of Nvidia Noise Removal while Monitoring is ON.

restested under these conditions, moving the slider with Monitoring on, but it works fine on my side.

**Comment by pkviet** [comment]
Try with a master build (artefacts are available for instance, here: https://github.com/obsproject/obs-studio/actions/runs/10732160058 bottom of page)
I've split the nvidia filters and fixed a bug when swapping effects. Maybe it'll work better in your case ?

**Comment by Sopheus** [comment]
> Try with a master build (artefacts are available for instance, here: https://github.com/obsproject/obs-studio/actions/runs/10732160058 bottom of page) I've split the nvidia filters and fixed a bug when swapping effects. Maybe it'll work better in your case ?

For some reason it does not start for me after I run it. And for some reason it takes 0% of CPU:

![image](https://github.com/user-attachments/assets/c718e64d-61d8-464c-93f7-3ad65c76f2c8)

**Comment by pkviet** [comment]
> > Try with a master build (artefacts are available for instance, here: https://github.com/obsproject/obs-studio/actions/runs/10732160058 bottom of page) I've split the nvidia filters and fixed a bug when swapping effects. Maybe it'll work better in your case ?
> 
> For some reason it does not start for me after I run it. And for some reason it takes 0% of CPU:
> 
mmh i have no idea what's going on then. Very sorry.
As a workaround, you could try using the nividia broadcast app , apply the filters there and select the nvidia broadcast driver as an obs audio source.

**Comment by Sopheus** [comment]
> > > Try with a master build (artefacts are available for instance, here: https://github.com/obsproject/obs-studio/actions/runs/10732160058 bottom of page) I've split the nvidia filters and fixed a bug when swapping effects. Maybe it'll work better in your case ?
> > 
> > 
> > For some reason it does not start for me after I run it. And for some reason it takes 0% of CPU:
> 
> mmh i have no idea what's going on then. Very sorry. As a workaround, you could try using the nividia broadcast app , apply the filters there and select the nvidia broadcast driver as an obs audio source.

Yeah, have tried with Broadcast app, same story unfortunately, not sure what causing this. But thank you for your time anyways.
